app/controllers/api/v1/pizzas_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::PizzasController < ApplicationController
+  def show
+    selected_pizza = Pizza.find(params[:id])
+    render json: {pizza: selected_pizza, brand: selected_pizza.brand} 
+  end
```

Comment:
please add spaces before and after curly brackets `{}` 

Additionally, I STRONGLY recommend adding serializers for this endpoint!